src/sql/workbench/contrib/executionPlan/browser/LongTextPopupViewer.ts

```diff
@@ -0,0 +1,126 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import 'vs/css!./media/LongTextPopupViewer';
+
+import * as DOM from 'vs/base/browser/dom';
+import { isString } from 'vs/base/common/types';
+import { editorBackground, editorForeground, textBlockQuoteBorder } from 'vs/platform/theme/common/colorRegistry';
+import { IColorTheme, ICssStyleCollector, registerThemingParticipant } from 'vs/platform/theme/common/themeService';
+
+export class LongTextPopupViewer<T extends Slick.SlickData> {
+
+	private _args: Slick.Editors.EditorOptions<T>;
+	private _wrapper: HTMLElement;
+	private _text: HTMLElement;
+
+	public constructor(args: Slick.Editors.EditorOptions<T>) {
+		this._args = args;
+		this.init();
+	}
+
+	public init(): void {
+		this._wrapper = DOM.$('.long-text-popup-viewer-wrapper');
+		this._wrapper.style.width = this._args.column.width + 'px';
+		this._text = DOM.$('span');
+		this._wrapper.tabIndex = 0;
+		this._wrapper.appendChild(this._text);
+		document.body.appendChild(this._wrapper);
+		this.position((<any>this._args).position);
+
+
+		this._wrapper.addEventListener('keydown', (e: KeyboardEvent) => {
+			let eventHandled = false;
+			if (e.key === 'Enter' || e.key === 'Escape') {
```

Comment:
Why does "enter" cancel changes?

Reply:
Because there are no changes in the first place. It is just a viewer